src/layers/__init__.py

```diff
@@ -1,5 +1,5 @@
-from add_norm import AddNorm
-from feed_forward import FeedForward
-from multihead import MultiHeadAttention
-from input_encoding import InputEncoding
-from output_embedding import OutputEmbedding
\ No newline at end of file
+from .add_norm import AddNorm
```

Comment:
Switching these to local imports because:
1. My env is fucking killing me
2. Chatgpt is able to argue that it is okay here